.env.sample

```
@@ -0,0 +1,13 @@
+DATABASE_URL="db_type://user_name:password@hostname:port/table_name"
+
+PORT = port
+
+TYPEORM_CONNECTION = db_type
+TYPEORM_HOST = host
+TYPEORM_USERNAME = user_name
+TYPEORM_PASSWORD = password
+TYPEORM_DATABASE = database_name
+TYPEORM_PORT = port
+TYPEORM_LOGGING = TRUE
```

대소문자 동일한 포맷으로 유지하면 좋을 것 같습니다!
```suggestion
TYPEORM_LOGGING = boolean
```